application-load-balancer/main.tf

```diff
@@ -1,3 +1,11 @@
+data "aws_ssm_parameter" "slack_token" {
+  name = "/codebuild/slack_oauth_token"
+}
+
+data "aws_ssm_parameter" "slack_alert_channel_alb_secret_rotation" {
+  name = var.config.slack_alert_channel_alb_secret_rotation
```

Comment:
If we make providing the slack parameter optional we need a default value for here. Although I'm not sure what we can use as a default channel.

Reply:
I would suggest the Alerts P2 as the default channel.

Reply:
I've added a default value for the Slack channel set to P2 alerts